cmd/zc_traverser_local.go

```diff
@@ -205,24 +215,36 @@ func WalkWithSymlinks(appCtx context.Context, fullPath string, walkFunc filepath
 
 				if err != nil {
 					WarnStdoutAndScanningLog(fmt.Sprintf("Failed to resolve symlink %s: %s", filePath, err))
+					if errorChannel != nil {
+						errorChannel <- ErrorFileInfo{FilePath: filePath, FileInfo: fileInfo, ErrorMsg: fileError}
+					}
```

Comment:
we should pass err instead of fileError (which is nil).
Better still we should pass a more descriptive error like

err = fmt.Errorf("Failed to resolve symlink %s: %s", filePath, err)

in all places